optimum/intel/__init__.py

```diff
@@ -94,6 +94,7 @@
             "OVModelForFeatureExtraction",
             "OVModelForImageClassification",
             "OVModelForMaskedLM",
+            "OVModelForPix2Struct",
```

Comment:
can you also add it here https://github.com/huggingface/optimum-intel/blob/ce6c6bc3bba2de86aa33be8da62290de94d9e938/optimum/intel/__init__.py#L188

Reply:
Could you also add it here as well 



https://github.com/huggingface/optimum-intel/blob/ce6c6bc3bba2de86aa33be8da62290de94d9e938/optimum/intel/openvino/utils.py#L87


https://github.com/huggingface/optimum-intel/blob/ce6c6bc3bba2de86aa33be8da62290de94d9e938/optimum/intel/utils/dummy_openvino_objects.py#L106

Reply:
Updated, thanks